source/features/load-hidden-comments.tsx

```diff
@@ -0,0 +1,50 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	// Retrieve the form associated with the "X Hidden Items\nLoad More" button
+	const form = select('form.ajax-pagination-form');
+	if (form) {
+		// The first button, which has the text "X hidden items".
+		const button = select('button', form);
+		if (button) {
+			// Extract the number of hidden items from the button text
+			const numberString = button.textContent?.trim()?.split(' ')[0];
+			if (numberString) {
+				// Construct the new URL. This is undocumented, and may break at any time.
+				// As of 12/08/2019, URLS look like this:
+				// "/_render_node/MDExOlB1bGxSZXF1ZXN0MjE2MDA0MzU5/timeline/more_items?variables%5Bafter%5D=Y3Vyc29yOnYyOpPPAAABZemjg2AAqTQyMjE5MTk1MQ%3D%3D&variables%5Bbefore%5D=Y3Vyc29yOnYyOpPPAAABaENrVHAAqTQ1Mzc3MjMzNg%3D%3D&variables%5Bfirst%5D=60&variables%5BhasFocusedReviewComment%5D=false&variables%5BhasFocusedReviewThread%5D=false"
+				// The key "variables[first]" in the URL query parameters appears to control
+				// how many additional comments are fetched.
+				// Github appears to always set this to 60, but it can be increased up to the
+				// total number of hidden items.
+				// By setting "variables[first]" to total number of hidden items (extracted from the button tex),
+				// we can fetch all hidden comments at once
+				const url = new URL('https://github.com' + (form.getAttribute('action')?.toString() || ''));
```

Comment:
Same here. Also we support GitHub Enterprise on non-`github.com` domains

```suggestion
				const url = new URL(form.action!, location.origin);
```